### ISSUE TITLE
User interface with buttons?

### ISSUE THREAD

**Issue by Theresa-S** [saalfeldlab/bigwarp#114]
This is more of an enhancement wish (I am using Big Warp in Fiji): All the shortcuts used for navigation and the commands are useful and well explained in the help page. Nevertheless, whenever some time had passed since I last used BigWarp, I need half a day to learn the shortcuts all over again. An additional user interface, where all main functions are available as buttons would therefore be really helpful in my opinion.

I'm aware that this is a bigger issue, which can't be implemented over night. Please regard this as feedback from a non-computer scientist end user.

**Comment by mattrussellcrick** [comment]
Agreed. Controls to set and recall view positions precisely would also be useful
